docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md

```diff
@@ -4,7 +4,7 @@ title: External DNS management - connection refused
 
 ## Symptom
 
-If you use a custom domain, you receive the `connection refused` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
+After all the steps required to prepare your custom domain are finished, you receive the `connection refused` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
```

Comment:
In the "Remedy" section (I can't comment there) I would add:

In addion, ensure that your OS resolves the target host name to the same Ingress Gateway IP address.
run:
```
   host httpbin,kyma-goat.ga
```

Reply:
`connection refused` or `connection timeout`